web/cobrands/fixmystreet-uk-councils/assets.js

```diff
@@ -592,6 +592,21 @@ fixmystreet.assets.centralbedfordshire.streetlight_stylemap = new OpenLayers.Sty
     'select': fixmystreet.assets.construct_named_select_style("${lighting_c}")
 });
 
+fixmystreet.assets.centralbedfordshire.style_default_green = new OpenLayers.Style({
```

Comment:
Note the distinction in this file between functions/variables that need to be referred to outside the file (e.g. by the asset layer configuration), so given externally-reference-able names under `fixmystreet.assets...` and ones that don't need to be referred to externally.